app/controllers/animal_sightings_controller.rb

```diff
@@ -1,53 +1,55 @@
 class AnimalSightingsController < ApplicationController
+      rescue_from ActiveRecord::RecordInvalid, with: :render_unprocessable_entity_response
+      rescue_from ActiveRecord::RecordNotFound, with: :render_not_found_response
+
+      def render_unprocessable_entity_response(exception)
+            render json: exception.record.errors, status: :unprocessable_entity
+      end
+  
+      def render_not_found_response(exception)
+            render json: { error: exception.message }, status: :not_found
+      end
```

Comment:
This may be a little excessive since validations will add the proper errors, and controllers will allow us to see them.